bin/start-dev

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+
+#
+# Render app-config.local.yaml from template
+#
+
+# Define the green and red color escape sequences
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+# Fallback to the second argument if the first is empty
+setf() {
+  if [ -z "${!1}" ]; then
+    export $1="$2"
+  fi
+}
+
+# Print key-value pair in green if value is not empty, otherwise in red
+printv() {
+  if [ -z "$2" ]; then
+    echo -e "${RED}$1${NC}=${RED}<empty>${NC}"
+  else
+    echo -e "${GREEN}$1${NC}=$2"
+  fi
+}
+
+# Print key-password pair in green if value is not empty, otherwise in red, replacing password with '*'
+printp() {
+  if [ -z "$2" ]; then
+    echo -e "${RED}$1${NC}=${RED}<empty>${NC}"
+  else
+    echo -e "${GREEN}$1${NC}=********"
+  fi
+}
+
+# Wait for process to finish while showing a spinner
+waitp() {
+  local pid=$1
+  local message=$2
+  local delay=0.1
+  local spinner_chars="/-\|"
+  echo -n "$messge"
+  while [ -d /proc/$pid ]; do
+    for i in $(seq 0 3); do
+      echo -ne "$message ${spinner_chars:i:1}" "\r"
+      sleep $delay
+    done
+  done
+
+  # Check the exit status of the process
+  wait $pid
+  local exit_status=$?
+
+  # Replace spinner with a check mark or an X based on the result
+  if [ $exit_status -eq 0 ]; then
+    echo -e "$message ${GREEN}✔${NC}"
+  else
+    echo -e "$message ${RED}✘${NC}"
+  fi
+}
+
+usage() {
+  echo "Usage: $0 [options]"
+  echo "Options:"
+  echo "  -t, --title <title>                   The title of the Backstage instance"
+  echo "  -o, --org <org>                       The organization name"
+  echo "  -u, --github-user <github-user>       The GitHub user"
+  echo "      --github-token <github-token>     The GitHub token"
+  echo "      --gitea-host <host>               The Gitea Host"
+  echo "      --gitea-user <gitea-user>         The Gitea username"
+  echo "      --gitea-password <gitea-password> The Gitea username"
```

Comment:
If the main goal is to use qshift and github, why do you still set gitea variables ?